conf/application.conf

```diff
@@ -54,7 +54,6 @@ play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModu
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.FrontendModule"
```

Comment:
change this package name to `uk.gov.hmrc.play.bootstrap.frontend.FrontendModule`